app/services/requisicion_service.py

```
@@ -0,0 +1,536 @@
+# app/services/requisicion_service.py
+
+"""
+Este módulo contendrá la lógica de negocio relacionada con las requisiciones.
+"""
+
+from flask import current_app, flash, abort
+from datetime import datetime
+import pytz
+
+from app import db, ESTADO_INICIAL_REQUISICION, ESTADOS_REQUISICION, ESTADOS_REQUISICION_DICT, ESTADOS_HISTORICOS, TIEMPO_LIMITE_EDICION_REQUISICION
```

Pylint (3.3.7)

[error] No name 'ESTADO_INICIAL_REQUISICION' in module 'app'

(E0611)